app/src/main/java/org/p2p/wallet/bridge/send/interactor/BridgeSendInteractor.kt

```diff
@@ -94,16 +95,59 @@ class BridgeSendInteractor(
         sendToBlockchain(feeRelayerTransaction.transaction)
     }
 
+    suspend fun transferFromSolana(
+        userWallet: SolAddress,
+        feePayer: SolAddress,
+        source: SolAddress,
+        recipient: EthAddress,
+        mint: SolAddress?,
+        amount: String
```

Comment:
it's not clear what exactly the "amount" is, please add comment and/or rename variable to something more obvious